spec/views/users/index_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+require 'capybara/rspec'
+
+RSpec.describe 'Users', type: :system do
+  before do
+    @user1 = User.create(name: 'Melanie', photo: '', bio: 'Hello my name is Melanie')
+    @user2 = User.create(name: 'Bianca', photo: '', bio: 'Hello my name is Bianca')
+  end
+
+  scenario 'Index' do
+    visit users_path
+    expect(page).to have_content(@user1.name)
+    expect(page).to have_css('img')
+    expect(page).to have_content("Number of posts: #{@user1.posts_counter}")
+    expect(page).to have_content(@user2.name)
+    expect(page).to have_content("Number of posts: #{@user2.posts_counter}")
+  end
+
+  scenario 'Redirects to show' do
+    visit users_path
+    click_link(@user1.name)
+    expect(page).to have_current_path(user_path(@user1.id))
+    expect(page).to have_content(@user1.name)
+    expect(page).to have_content("Number of posts: #{@user1.posts_counter}")
+  end
```

Comment:
- [ ] @BiancaMe Team Great work adding tests for the user index page, however the tests should be checking if all users image username  number of posts and link are working, check the example below on how you can implement it in your project
```suggestion
 scenario 'Index' do
    visit users_path
    @users.each do |user|
      expect(page).to have_content(user.name)
      expect(page).to have_css('img')
      expect(page).to have_content("Number of posts: #{user.posts_counter}")
    end
  end
```